superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -853,12 +864,16 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     }).then(response => setChartVizType(response.json.result.viz_type));
 
   // Handle input/textarea updates
-  const onTextChange = (
+  const onInputChange = (
     event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
   ) => {
     const { target } = event;
+    const value =
+      target.type === 'number'
```

Comment:
is there an enum we can attach to this?

Reply:
No, it's just the `<input type="XXX" />` value.